content/en/docs/concepts/security/hardening-guide/scheduler.md

```diff
@@ -0,0 +1,92 @@
+| `authentication-kubeconfig`              | Make sure to provide a proper kubeconfig so that the server calls are secure. This kubeconfig file should also be maintained securely. |
+| `authentication-tolerate-lookup-failure` | Set to `false` to make sure invalid authentication configurations do not lead to requests passing off as anonymous                     |
+| `authentication-skip-lookup`             | This should be set to `false` to make sure all missing authentication configuration falls back to the authentication kubeconfig.       |
+| `authorization-always-allow-paths`       | These paths should respond with data that is appropriate for anonymous authorization. Defaults to `/healthz,/readyz,/livez`.           |
+{{</table>}}
+
+### Scheduler networking command line options
+{{<table caption="Security advice for kube-scheduler command line options relating to networking" >}}
+| Command line argument    | Security hardening advice                                                                                                                                            |
+| ------------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `bind-address`           | In most cases, the kube-scheduler does not need to be externally accessible. Setting the bind address to `localhost` is a secure practice.                           |
+| `permit-address-sharing` | Set this to `false` to  disable connection sharing through `SO_REUSEADDR`. `SO_REUSEADDR` can lead to reuse of terminated connections that are in `TIME_WAIT` state. |
+| `permit-port-sharing`    | Default `false`. Use the default unless you are confident you understand the security implications.                                                                  |
+{{</table>}}
+
+### Scheduler TLS command line options
+{{<table caption="Security advice for kube-scheduler command line options relating to encryption in transit" >}}
+| Command line argument          | Security hardening advice                                                                                                                                                                                                          |
+| ------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `requestheader-client-ca-file` | Generally do not depend on authorization being already done for incoming requests. Always provide the root certificate bundle. This allows authorization to happen on each incoming request through `requestheader-allowed-names`. |
```

Comment:
I think setting this without setting all the other requestheader* configuration options may actually be very dangerous. I've started a thread with you on slack.

/hold